test_fixtures.py

```diff
@@ -1,5 +1,144 @@
 import maas_status_enum
 
+test_fetch_machines_grouped_by_tags_tags = [
+    {
+        'name': 'ceph'
```

Comment:
always end with a `,` even if it's the last element. the reason for this is, that if you're able to do it (in this case you are) it will make the diff cleaner

imagine that you add a new property in this object, the diff will show 2 lines and the only thing that matters is the property you've added

```
# diff when you need to add the new line

+ 'name': 'ceph',
+ 'new-property': 'shit'

```
compared to

```
# diff when you don't need to add the new line

+ 'new-property': 'shit'

```